src/main/kotlin/org/b12x/gfe/core/controller/version/VersionList.kt

```diff
@@ -0,0 +1,42 @@
+package org.b12x.gfe.core.controller.version
+
+import org.b12x.gfe.core.model.parsers.ParserVersionData
+import org.b12x.gfe.utilities.DirectoryManagement
+import org.b12x.gfe.utilities.FileManagement
+
+class VersionList(loci: String) {
+    private val localVersions = LocalVersions(loci)
+    private val onlineVersionNames = getOnlineVersionList()
+    var allVersionNames = combineVersionLists().sortedDescending()
+    val availableVersions = ArrayList<String>()
+
+    private fun getOnlineVersionList(): ArrayList<String> {
+        val versionList = ArrayList<String>()
+        val directoryManagement = DirectoryManagement()
```

Comment:
You might as well comment this out too.

Reply:
Same as above, the class is incomplete. I will deal with this when I return to that class.